.github/workflows/all_green_ckeck.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: all_green
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+    branches:
+      - main
+      - 'stable-*'
+    tags:
+      - '*'
+
+jobs:
+  changelog-and-linters:
+    uses: ./.github/workflows/changelog_and_linters.yml  # use the callable changelog_and_linters job to run tests
+  linters:
+    uses: ./.github/workflows/linters.yml  # use the callable linters job to run tests
```

Comment:
This workflow is missing. This line has to be removed or linters.yml should be added.